intelmq/bots/outputs/rpz_file/output.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+import datetime
+from datetime import datetime
+import os
+from collections import defaultdict
+from pathlib import Path
+
+from intelmq.lib.bot import OutputBot
+
+RPZ_INDICATOR_MAP = {
+    "source.fqdn": "Intel::DOMAIN"
+}
+
+now = datetime.now()
+
+
+class RpzFileOutputBot(OutputBot):
+    _file = None
+    format_filename: bool = False
+    hierarchical_output: bool = False
+    keep_raw_field: bool = False
+    message_jsondict_as_string: bool = False
+    message_with_type: bool = False
+    single_key: bool = False
+    is_multithreadable = False
```

Comment:
That should work as intended:
```suggestion
    __Bot_is_multithreadable = False
```
(but I see that using a private member was not the best choice for this, as this requires the special naming in cases like this one - but that's a different discussion)